scripts/components/App.js

```diff
@@ -60,8 +60,15 @@ export class Store extends EventEmitter {
       .then(res => {
         this.state.items.push(res.data);
         this.emit('change', this.state);
-      })
```

Comment:
It's probably good to keep catching errors. Maybe we could trigger an error event with the details, so subscribed components could render a fancy flash message or something?